includes/wc-template-functions.php

```diff
@@ -2546,17 +2546,43 @@ function wc_get_stock_html( $product ) {
  *
  * @since  3.0.0
  * @param  float $rating Rating being shown.
+ * @param  int   $count  Total of ratings.
```

Comment:
"Total number of ratings." makes more sense. Otherwise it sounds like `$count` is the total of all the ratings added together. Everything else looks good.